pkg/redact/redact.go

```diff
@@ -42,6 +44,28 @@ func GetRedactSpec() (string, string, error) {
 	return encodedData, "", nil
 }
 
+func GetRedact() (*v1beta1.Redactor, error) {
+	spec, _, err := GetRedactSpec()
+	if err != nil {
+		return nil, err
+	}
+	if spec == "" {
+		return nil, nil
+	}
+
+	scheme.AddToScheme(scheme.Scheme)
```

Comment:
can you move this to an init function?

Reply:
do we do this in an init function anywhere else?

Reply:
we should. ive mentioned it before. this is how they do it in the k8s codebase
https://github.com/kubernetes/kubernetes/blob/0e7376d9cb1ec74bcb3c10a664a330bc9d60df64/staging/src/k8s.io/metrics/pkg/client/clientset/versioned/scheme/register.go#L57

Reply:
https://godoc.org/sigs.k8s.io/controller-runtime/pkg/scheme

Reply:
in any case it cant be in concurrent code. we could add it to the main or to an initialization function

Reply:
main sounds like a better location then
to make sure multiple inits don't run at once

Reply:
though this _is_ a different scheme package